buddyboss/buddyboss-platform
# Wrong use of 'an' when you rename 'Organizer' group role inside 'Group Types' settings.

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #602.

### How to test the changes in this Pull Request:
1. Go to BuddyBoss -> groups -> group types. 
2. Create a group type. 
3. Rename the 'Organizer' group role to 'Teacher'.
4. Assign a group to this 'Group Type'.
5. Go to 'Groups' directory page.
6. See the text on the assigned group. 

### Proof Screenshots or Video
http://somup.com/cYnu1thtbw


### Other information:
